docs/installation/mac.md

```diff
@@ -49,135 +41,228 @@ Docker container using standard localhost addressing such as `localhost:8000` or
+### From the Docker Quickstart Terminal
 
-1. Create a new Boot2Docker VM.
+1. Open the "Applications" folder or the "Launchpad".
 
-        $ boot2docker init
+2. Find the Docker Quickstart Terminal  and double-click to launch it.
 
-    This creates a new virtual machine. You only need to run this command once.
+    The application:
 
-2. Start the `boot2docker` VM.
+    * opens a terminal window
+    * creates a VM called `docker-vm` if it doesn't exists, starts the VM if it does
+    * points the terminal environment to this VM
 
-        $ boot2docker start
+    Once the launch completes, the Docker Quickstart Terminal reports:
+    
+     ![All finished](/installation/images/mac-success.png)
+     
+    Now, you can run `docker` commands. 
 
-3. Display the environment variables for the Docker client.
+3. Verify your setup succeeded by running the `hello-world` container.
 
-        $ boot2docker shellinit
-        Writing /Users/mary/.boot2docker/certs/boot2docker-vm/ca.pem
-        Writing /Users/mary/.boot2docker/certs/boot2docker-vm/cert.pem
-        Writing /Users/mary/.boot2docker/certs/boot2docker-vm/key.pem
-            export DOCKER_HOST=tcp://192.168.59.103:2376
-            export DOCKER_CERT_PATH=/Users/mary/.boot2docker/certs/boot2docker-vm
-            export DOCKER_TLS_VERIFY=1
-
-    The specific paths and address on your machine will be different.
-
-4. To set the environment variables in your shell do the following:
-
-        $ eval "$(boot2docker shellinit)"
-
-    You can also set them manually by using the `export` commands `boot2docker`
-    returns.
+        $ docker run hello-world
+        Unable to find image 'hello-world:latest' locally
+        511136ea3c5a: Pull complete
+        31cbccb51277: Pull complete
+        e45a5af57b00: Pull complete
+        hello-world:latest: The image you are pulling has been verified.
+        Important: image verification is a tech preview feature and should not be
+        relied on to provide security.
+        Status: Downloaded newer image for hello-world:latest
+        Hello from Docker.
+        This message shows that your installation appears to be working correctly.
+
+        To generate this message, Docker took the following steps:
+        1. The Docker client contacted the Docker daemon.
+        2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
+           (Assuming it was not already locally available.)
+        3. The Docker daemon created a new container from that image which runs the
+           executable that produces the output you are currently reading.
+        4. The Docker daemon streamed that output to the Docker client, which sent it
+           to your terminal.
+
+        To try something more ambitious, you can run an Ubuntu container with:
+        $ docker run -it ubuntu bash
+
+        For more examples and ideas, visit:
+        http://docs.docker.com/userguide/
```

Comment:
Bah, looks like the image still isn't updated on docker hub. I created https://github.com/docker-library/hello-world/commit/22ecfe456f254d5babe6e413bed2de77cfaba047

@tianon ? :heart: